pkg/scheduler/partition.go

```diff
@@ -196,6 +196,7 @@ func (pc *PartitionContext) addQueue(conf []configs.QueueConfig, parent *objects
 		if err != nil {
 			return err
 		}
+		thisQueue.SendNewQueueEvent()
```

Comment:
I am not really sure about triggering new queue event from this place as there are some other places where this needs to be triggered. Instead of doing this in 'n' number of places, I think it is better to trigger this even after queue creation in corresponding constructors in `Queue` itself. For example, NewConfiguredQueue and NewDynamicQueue

Reply:
I was thinking about this before, it seemed to be a better idea because it makes testing slightly easier (no extra event when creating queues). I can check it and see how much changes is needed.

Reply:
Ok, see if you can address now or file a new jira